### ISSUE TITLE
Search: Checkbox is not consistently identified in the note

### ISSUE THREAD

**Issue by sarasuri** [mmiron-HC/WST-RSAMS#158]
Page: Search
URL: https://rsams.dev.cloud.openplus.ca/en/search/site
User Agent: Firefox (v99)
Language: English and French

Issue Description: In the note below, checkbox is not consistently identified. The reference to the checkbox in the not the same as the its label text.

![image](https://user-images.githubusercontent.com/12256067/162065765-5a3563f3-70fc-41c7-aa85-7044e1a9016b.png)

Steps to reproduce:

1. Visual scanning

Expected results: The checkbox should be identified consistently in the note using its actual label text.

**Comment by davidgotlieb** [comment]
@echouin Can you work with OP to adjust the wording on the search page to match the text of the check box?

**Comment by echouin** [comment]
No longer an issue as text is now removed.

**Comment by 40999221** [comment]
The layout has been changed. "Note" was removed. Issue no longer exist.

![image](https://github.com/user-attachments/assets/a78cdbcd-6147-4c01-ac17-dc5e9c87871f)

**Comment by zimw7** [comment]
Issue no longer exists. Layout has changed and the note has been removed.
![image](https://github.com/user-attachments/assets/8e80ec77-a6c7-447c-a2e1-512962355d82)

**Comment by YuxuanXie01** [comment]
The issue is no longer present since the note section has been removed.
